### ISSUE TITLE
Opt in tracking

### ISSUE THREAD

**Issue by johlrogge** [agical/friendsunhosted#25]
I am really curious to know how many are actually using F#Un. I would not want to force users to be tracked but it would be nice if users would allow some statistics to be gathered. What if the users were presented with a question upon login (once only)

"Thanks for using friends unhosted. We wouldn't dream of tracking your behavior. However, we /are/ curious about how the network grows and how many are using it. Would you like to send an anonymous tick for our statistics? Yes/No (If you really like what we are doing here, please consider flattering, sending bitcoins or make a donation via paypal)"

Then when we release new features and stuff we can allow users to give us a thumbs up/down for the funtionality.

The data could be gathered in google analytics or whichever service we prefer. The important thing is: tracking is opt in. We can track what people like or not but only if they chose to let us know.

**Comment by jancborchardt** [comment]
Yes, opt-in tracking sounds good. But even when opted in, the data should probably be gathered by a [Piwik](http://piwik.org) instance instead of giving it to Google.

**Comment by johlrogge** [comment]
Agree. Or maybe set up a graphite-instance? cc @codification

**Comment by brolund** [comment]
-1 for anything Google. :-) Otherwise, I'm also very curious.

**Comment by codification** [comment]
Setting up Graphite with some kind of HTTP frontend should be straightforward. Perhaps as a sudbdomain to friendsunhosted.com?
If we stick to just opt-in and just aggregating numbers (and showing them openly) there should be no problems with integrity as far as I can se.

**Comment by jancborchardt** [comment]
Yeah, subdomain for that sounds good. In addition to the realtime graphing of Graphite it would be cool to have [Piwiki](http://piwik.org) for stuff like bounce rate, referers etc.

**Comment by johlrogge** [comment]
I really like showing the data openly. Piwiki seems to solve a different, also interesting problem. All in all they seem complementary

**Comment by jancborchardt** [comment]
Open stats are really nice! For ownCloud we have Piwik set up openly at http://stats.owncloud.org – displaying all the visitor data to http://ownCloud.org and the demo at http://demo.ownCloud.org

**Comment by johlrogge** [comment]
Wow, piwik looks F#n awesome!
